<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>finalproject</title>
    <meta name="author" content="Adtile">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link rel="stylesheet" href="responsive-nav-files/css/styles.css">
    <link rel="stylesheet" href="css/styles.css">
    <script src="responsive-nav-files/js/responsive-nav.js"></script>
    <style>
    @import url('https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Roboto:ital,wdth@0,75;1,75&display=swap');
    </style>
</head>

<body>
    <header>
        <div class="header-content">
        <div class="logo">
            <a href="index.html"><img src="img/hicks_logo.png" alt="Hicks lab logo"></a>
        </div>
        <nav class="nav-collapse inter"> 
            <ul>
                <li class="menu-item"><a href="index.html" data-scroll>HOME</a></li>
                <li class="menu-item"><a href="people.html" data-scroll>PEOPLE</a></li>
                <li class="menu-item"><a href="research.html" data-scroll>RESEARCH</a></li>
                <li class="menu-item"><a href="plants.html" data-scroll>PLANTS</a></li>
                <li class="menu-item"><a href="proteomics.html" data-scroll>PROTEOMICS</a></li>
            </ul>
        </nav>
        </div>
        <script src="responsive-nav-files/js/fastclick.js"></script>
        <script src="responsive-nav-files/js/scroll.js"></script>
        <script src="responsive-nav-files/js/fixed-responsive-nav.js"></script>
    </header>

<main>
    <article>
        <div class="heading no-margin">
            <h1 class="inter">Proteomics Signaling Networks</h1>
        </div>
        <p class="roboto">Our laboratory has developed quantitative proteomics approaches for systems biology analyses in green algae, plant and microbial systems. Due to their critical roles in regulation of cellular signaling, we have focused on approaches to explore proteome-wide post-translational modifications such as reversible oxidation, phosphorylation, and proteolysis.</p>
        <div class="img-container">
            <img src="img/signaling.PNG" alt="Diagram outlining how tardigrades accumulate ROS following stress exposure ">
        </div>
    
        <div class="heading small-margin">
            <h2 class="inter small-margin">Thiol-based Regulatory Switches</h2>
        </div>
        <p class="roboto">The influence of the redox environment as a regulatory factor in cellular processes has been present since the origin of life but is only beginning to be examined in detail. With a focus on the identification of protein targets of oxidation, the respective modification type, site specificity, quantification, and functional consequence, the field of redox biology remains highly complex and requires multiple approaches for systematic characterization. We routinely implement global and site-specific redox proteomics characterization methods by coupling an optimized reproducible disulfide-bond exchange resin enrichment (TPS6B) at the peptide- or protein-level with label-free quantitation via LC-MS/MS to elucidate relative redox changes in complex protein mixtures. We have coupled this with other PTM enrichments to facilitate the identification and elucidation of crosstalk between essential signaling pathway networks. Identification of redox proteins serves as a crucial element in understanding stress response in photosynthetic organisms, tardigrades and beyond.</p>
        <div class="heading small-margin">
            <h2 class="inter small-margin">Phosphoproteomics</h2>
        </div>
        <p class="roboto">Protein phosphorylation, encompassing the addition or removal of phosphate moieties on the functional groups of amino acid side chains (typically Ser, Thr, Tyr), is ubiquitously found in eukaryotes and revealed to be critical for growth, adaptation and survival in photosynthetic organisms. Catalyzed by a class of enzymes known as kinases, these phosphorylation events serve as activators of protein function within the cell through induction of conformational change, and these kinases form networks that propagate phosphorylation signals through specific biological pathways as a mechanism for generating cellular responses from various stimuli.</p>
        <p class="roboto">The study of these phosphorylation events in the cellular proteome, or phosphoproteomics, in photosynthetic organisms has the potential to not only inform on how life is sustained but also how an organism responds to their environment, with insights into broad ranging fields from agriculture, biofuels, and medicine. Our lab seeks to elucidate these phosphorylation events through complementary in vivo and in vitro platforms that utilize enrichment strategies specific to phosphorylation coupled with liquid chromatography and high mass accuracy, high resolution mass spectrometry (LC-MS) for broad, quantitative coverage of the phosphoproteome. Using enrichment strategies coupled with label-free quantitative proteomics, we have deeply interrogated the dynamic proteome and phosphoproteome of the model algae Chlamydomonas reinhardtii with >13 publications.</p>
        <div class="heading small-margin">
            <h2 class="inter small-margin">Proteolytic Pathways Involved in Stress Signaling Responses</h2>
        </div>
        <p class="roboto">Plants are dynamic systems wherein external and internal signals are integrated to facilitate and sustain life. The intricate balance of protein translation and degradation, known as protein homeostasis, is key to maintaining proper cellular function. Despite its unequivocal importance, plant protein homeostasis and its intersection with stress signaling is relatively unexplored. Furthering our understanding of proteolysis in the plant stress response could impact the development of more adaptive and resistant crops.</p>
        <div class="heading small-margin">
            <h2 class="inter small-margin">Microbiome</h2>
            <p class="roboto">The role of the microbiome in human health and disease is well documented. Commensal bacteria produce complex metabolites to compete or protect against neighboring species; however, their roles in niche protection and host physiology, susceptibility to bioactive peptides introduced through diet, and connections between intra-species interactions are nascent.</p>
        </div>
</main>

<footer>
    <div class="footer-content">
    <div class="slogan">
        <img src="img/hicks_logo_white.png" alt="Hicks lab logo">
        <p class="roboto white">Research in the Hicks lab focuses on development and implementation of mass spectrometric approaches for protein characterization including post-translational modifications, as well as the identification of bioactive peptides/proteins from plants.</p>
    </div>
    <div class="bottom-footer">
        <div class="footer-img">
            <img src="img/hicks_footer.png" alt="HICKS peptide chain">
        </div>
        <div class="contact">
            <h2 class="inter white">Contact Us</h2>
            <p class="roboto">Caudill 339</p>
            <p class="roboto">131 South Road, CB#3290</p>
            <p class="roboto">Chapel Hill, NC 27599</p>
            <p class="roboto">919-843-6903</p>
            <a class="roboto email white" href="mailto:lmhicks@unc.edu">lmhicks@unc.edu</a>
        </div>
    </div>
    </div>
</footer>
</body>
</html>